src/app/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -132,10 +140,89 @@ export class CheckoutOrderSummary extends PureComponent {
         );
     }
 
-    renderTotals() {
+    renderSubPrice(price) {
+        const {
+            totals: {
+                quote_currency_code
+            }
+        } = this.props;
+
+        if (!price) {
+            return null;
+        }
+
+        return (
+            <span>
+                { `${ __('Excl. tax:') } ${ formatPrice(price || 0, quote_currency_code) }` }
```

Comment:
Please convert `price || 0` to default function's argument value

Reply:
Done